corehq/ex-submodules/pillowtop/tests/test_bulk.py

```diff
@@ -1,5 +1,6 @@
 # coding=utf-8
 from __future__ import absolute_import
+from __future__ import unicode_literals
```

Comment:
Test failure real, but I don't understand it.

It's failing on the first of the four generated cases. If you have 10 bulk changes, your max size is 100, and your chunk size is 1, then you SHOULD get 10 payloads, not 3, which is the value given for "`expected_payloads`". so the test is failing, but the actual result is what I would expect ... right? :confused:

I also can't see how the failure has anything to do with `unicode_literals`.

I'll kick Travis, just in case the matrix is glitching.